bookkeeper-server/src/test/java/org/apache/bookkeeper/test/MultiLedgerManagerMultiDigestTestCase.java

```diff
@@ -48,6 +48,7 @@ public MultiLedgerManagerMultiDigestTestCase(int numBookies) {
     public static Collection<Object[]> configs() {
         String[] ledgerManagers = {
             "org.apache.bookkeeper.meta.FlatLedgerManagerFactory",
+            "org.apache.bookkeeper.meta.LegacyHierarchicalLedgerManagerFactory",
             "org.apache.bookkeeper.meta.HierarchicalLedgerManagerFactory",
             "org.apache.bookkeeper.meta.LongHierarchicalLedgerManagerFactory",
```

Comment:
Having this here is still causing tests to fail.  We should disable it for BookieRecoveryTest.ensurePasswordUsedForOldLedgers, because the consensus was that we want to keep the two ledger managers separate.